nil/core-concepts/sequencing.mdx

```diff
@@ -0,0 +1,131 @@
+# Sequencing
+
+## Definition
+
+As =nil; adopts the proposer-builder separation (PBS) model, the key principle of its approach to sequencing lies in building a network of builders and searchers competing for maximum extractable value (MEV) in relay auctions.
+
+:::tip[Justification]
+
+The sequencing structure of =nil; is fully compatible with Ethereum, which means that builders and searchers from L1 can easily participate in =nil;. This boosts protocol stability and liveness without sacrificing sovereignty.
+
+:::
+
+## Sequencing at the execution shard level
+
+PBS participants can freely choose the shard that they want to interact with. 
+
+While there exists a theoretical risk of nodes deciding to only partner with shards that host high-gain dApps (such as decentralized exchanges), this risk is negligible for two reasons.
+
+As the number of effective builders in a shard grows, the chances of attaining rewards for participants decreases:
+
+$$
+E(G) = 1/n * p
+$$
+
+, where $n$ is the number of effective builders, and $p$ is the expected gain.
+
+:::info[Example]
+
+Consider another shard where the competition is less intense and $k$ is the expected gain so that: 
+
+$$
+n \lt p/k
+$$
+
+Despite the gain itself being lower, the chances of attaining it are higher in this shard compared to the highly competitive shard described previously.
+
+$$
+E(G_k) \gt E(G_n)
+$$
+
+:::
+
+Additionally, execution shards are merged if their builders are inactive and their block count is below half of the block capacity of these shards. This prevents another possible exploit where a participant may specifically target underutilized shards to attain rewards reliably.
+
+:::note[Merge events]
```

Comment:
Should be updated according to the last design changes

Reply:
removed merge events from this page

Reply:
also removed PBS in favour of collator